.gitignore

```diff
@@ -1,7 +1,6 @@
 ## Directorties
 node_modules
 coverage/
-dist/
```

Comment:
Could you remove this change?

Reply:
Oops, didn't mean to leave that in, will remove it when I'm back in the office tomorrow!

Reply:
Done.

Reply:
Could you also remove the checked-in dist files below? Then I think it's fine to merge